DotNet/proxy.ashx

```diff
@@ -1,4 +1,4 @@
-<%@ WebHandler Language="C#" Class="proxy" %>
+<asp:TreeView runat="server"></asp:TreeView><%@ WebHandler Language="C#" Class="proxy" %>
```

Comment:
@markdeaton - not sure why you added asp:TreeView, but the PR seems to work fine without it, so I'll remove it before merging.